zerver/migrations/0328_migrate_to_edit_topic_policy.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.2 on 2021-05-26 09:43
+
+from django.db import migrations
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def migrate_to_edit_topic_policy(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    Realm = apps.get_model("zerver", "Realm")
+    Realm.POLICY_EVERYONE = 5
+    Realm.POLICY_ADMINS_ONLY = 2
+    Realm.objects.filter(allow_community_topic_editing=False).update(
+        invite_to_realm_policy=Realm.POLICY_ADMINS_ONLY
+    )
+    Realm.objects.filter(allow_community_topic_editing=True).update(
+        invite_to_realm_policy=Realm.POLICY_EVERYONE
+    )
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zerver", "0327_realm_edit_topic_policy"),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            migrate_to_edit_topic_policy, reverse_code=migrations.RunPython.noop, elidable=True
+        ),
```

Comment:
Generally, we prefer to write `reverse_code` for migrations like this to copy the data back, since without that one loses data if one reverses the migration.